### PR TITLE
Readme fix image shift

### PR DESCRIPTION
**Pull request by sambernaerdt**
Added a guide to the readme on how to fix an image shift in Photoshop.

### CODE REVIEW

File: guides/fix_image_shift.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+The image where the shift happened is called *image 2* in this guide, the one before it *image 1*
+Using Photoshop, fix as follows.
+
+#### Fix image 2
+
+* Open image 2
+
+  ![step1](fix_image_shift_step_1.png)
+  
+* Place image 1 on top and **transform** it **50%, 50%**
```
Comment (mwydmuch) [inline guides/fix_image_shift.md]: I think here we need a comment that 50%, 50% is for zoom factor = 2, the general formula for this transform should be 1 / zoom.

**Review by mwydmuch — APPROVED** [review]

**Comment by mwydmuch** [comment]
Thank you, this is a nice tutorial! I've made some minor edits, please see if you like them, and I will merge the PR.

**Comment by sambernaerdt** [comment]
Yes, perfect! Good edits. Thanks.

**Comment by mwydmuch** [comment]
Thank you! Merged!

**Pull request merged**